README.md

```diff
@@ -1731,6 +1732,14 @@ _See also: [awesome-sysadmin/Code Review](https://github.com/awesome-foss/awesom
 - [UVDesk](https://www.uvdesk.com/) - UVDesk community is a service oriented, event driven extensible opensource helpdesk system that can be used by your organization to provide efficient support to your clients effortlessly whichever way you imagine. ([Demo](https://demo.uvdesk.com/), [Source Code](https://github.com/uvdesk/community-skeleton)) `MIT` `PHP`
 - [ZenTao](https://www.zentao.pm/) - An agile(scrum) project management system/tool. ([Demo](https://demo.zentao.pm/user-login.html), [Source Code](https://github.com/easysoft/zentaopms)) `ZPL-1.2` `PHP`
 
+### Software Development - Security & QA
+
+**[`^        back to top        ^`](#)**
+
+- [Mega-Linter](https://megalinter.github.io/latest) - Mega-Linter can handle any type of project thanks to its 70+ embedded Linters, its advanced reporting, runnable on any CI system or locally, with assisted installation and configuration, able to apply formatting and fixes. `AGPL-3.0` `Python`
+- [Scanmycode CE (Community Edtion)](https://www.scanmycode.today/) - Code Scanning/SAST/Static Analysis/Linting using many tools/Scanners with One Report - Scanmycode Community Edition (CE). ([Source Code](https://github.com/marcinguy/scanmycode-ce)) `BSD-3-Clause LGPL-2.1 Commonsclause` `Python`
```

Comment:
Unfortunately scanmycode is non-free: https://github.com/marcinguy/scanmycode-ce/blob/master/LICENSE

> the grant of rights under the License will not include, and the License does not grant to you, the right to Sell the Software.

Please move it to non-free.md

Reply:
ping @marcinguy can you please move scanmycode CE to `non-free.md` (or remove it completely)?

Reply:
@nodiscc Moved it to `non-free.md`

Well, it is actually free to use (also commercial use, in companies) and Open Source, but not if you make money on it (to prevent abuse from big corps). Then you need to talk to me first.

Hope that clears it up.

Maybe it can be in both with the note above?

Reply:
I understand that, but it also goes against freedom 0 of [Free Software](https://en.wikipedia.org/wiki/Free_software):

> The freedom to use the program for any purpose.

(regardless of good intentions) That is why we also have [CC-BY-NC](https://en.wikipedia.org/wiki/Creative_Commons_NonCommercial_license) and similar "no commercial/production use" licences in `non-free.md`. See also [DFSG](https://en.wikipedia.org/wiki/Debian_Free_Software_Guidelines)

Thanks for the update